pkg/apis/helm.fluxcd.io/v1/types.go

```diff
@@ -89,6 +89,23 @@ type ValuesFromSource struct {
 	ChartFileRef *ChartFileSelector `json:"chartFileRef,omitempty"`
 }
 
+type NamespacedConfigMapKeySelector struct {
+	corev1.LocalObjectReference `json:",inline" protobuf:"bytes,1,opt,name=localObjectReference"`
+	Key                         string `json:"key" protobuf:"bytes,2,opt,name=key"`
+	// +optional
+	Optional *bool `json:"optional,omitempty" protobuf:"varint,3,opt,name=optional"`
+	// +optional
+	Namespace string `json:"optional,omitempty" protobuf:"bytes,4,opt,name=namespace"`
+}
+type NamespacedSecretKeySelector struct {
+	corev1.LocalObjectReference `json:",inline" protobuf:"bytes,1,opt,name=localObjectReference"`
```

Comment:
```suggestion
	Name string `json:"name"`
```
Please remove all protobuf annotations, we don't use those in CRDs